EarlGrey/Common/GREYAnalytics.h

```diff
@@ -16,7 +16,7 @@
 
 #import <Foundation/Foundation.h>
 
-@protocol GREYAnalyticsDelegate;
+#import "Common/GREYAnalyticsDelegate.h"
```

Comment:
#import <EarlGrey/GREYAnalyticsDelegate.h>
we do similar in rest of the headers as well.